Makefile

```diff
@@ -11,7 +11,11 @@ TIMEOUT := 60
 UNION := functional docker crio docker-compose network netmon docker-stability oci openshift kubernetes swarm vm-factory entropy ramdisk shimv2
 
 # skipped test suites for docker integration tests
-SKIP := $(shell bash -f .ci/hypervisors/$(KATA_HYPERVISOR)/filter_docker_$(KATA_HYPERVISOR).sh)
+ifeq ($(KATA_HYPERVISOR),)
+	SKIP :=
+else
+	SKIP := $(shell bash -f .ci/hypervisors/$(KATA_HYPERVISOR)/filter_docker_$(KATA_HYPERVISOR).sh)
```

Comment:
Oh, OK. I was imagining we might do an 'if-exists' on the actual destination file to check it existed before we tried to run it. That way, if KATA_HYPERVISOR is set, but the file does not exist, we would still silently ignore it. With the above, I think if KATA_HYPERVISOR is set, but the relevant file does not exist in the repo, we will still get the error won't we?